config/environments/production.rb

```diff
@@ -55,15 +55,19 @@
   # config.cache_store = :mem_cache_store
 
   # Use a real queuing backend for Active Job (and separate queues per environment).
-  # config.active_job.queue_adapter     = :resque
+  config.active_job.queue_adapter = :sidekiq
```

Comment:
Was this necessary?

It seems like we have code that does the same in `config/initializers/active_job.rb`.

Reply:
It is not necessary, I added this thinking that it might be sidekiq deliver later issue that mails were not sending.